src/views/certifications/views/[slug]/components/program-hero/index.tsx

```diff
@@ -0,0 +1,29 @@
+import classNames from 'classnames'
+// Shared under certifications
+import { CertificationsHero } from 'views/certifications/components'
+import { ProgramSlug } from 'views/certifications/types'
+// Styles
+import s from './program-hero.module.css'
+
+function ProgramHero({
+	heading,
+	description,
+	slug,
+}: {
+	heading: string
+	description: string
+	slug: ProgramSlug
+}) {
+	return (
+		<CertificationsHero
+			heading={heading}
+			description={description}
+			foreground={slug === 'security-automation' ? 'dark' : 'light'}
```

Comment:
💭 Future thought: I wonder if this setting could be data driven, in case we update the backgrounds in the future. Maybe we could use an enum or const above in the short term?

```
enum ProgramForeground {
	'security-automation' = 'dark',
  //... etc
}
```

Reply:
We could also set a default in case a new slug type is introduced?

Reply:
I dig this! 💯 Updated with an `enum` as suggested, and added a lil docs comment on how the `foreground` and `theme-<slug>` background classes relate.

We have some defaults within `CertificationsHero` as well - dark text, white background. So in the case a new program page with a new`slug` is added, the hero will default to something like this:

<img width="1506" alt="CleanShot 2022-12-14 at 11 35 14@2x" src="https://user-images.githubusercontent.com/4624598/207654367-8d2fa17c-c333-4530-84b8-a805ca2f525b.png">

Which obvs means we'll want to remember to add more background images to this component if we add a new page... but the default will at least kind of work & hit the color contrast requirements 👍

Reply:
Ooh came up with something slightly different - rather than the enum, I've now got this in place:

```
const programHeroForeground: Record<
	ProgramSlug,
	CertificationsHeroProps['foreground']
> = {
	'security-automation': 'dark',
	'infrastructure-automation': 'light',
	'networking-automation': 'light',
}
```

This should ensure that if a `ProgramSlug` is added, we'll get a TypeScript error unless we've explicitly handled the new `ProgramSlug` within this `ProgramHero` component.